pymongo/topology.py

```diff
@@ -890,7 +890,7 @@ def __repr__(self) -> str:
             msg = "CLOSED "
         return f"<{self.__class__.__name__} {msg}{self._description!r}>"
 
-    def eq_props(self):
+    def eq_props(self) -> Tuple[Tuple[Tuple[str, int], ...], Optional[str], Optional[str], str]:
```

Comment:
Probably can be `Tuple[Any]`.

Reply:
It's probably better to have the specifics; replaced the innermost `Tuple[str, int]` with `_Address`, which is probably more accurate.